spec/requests/user/user_suggestions_spec.rb

```diff
@@ -56,20 +59,31 @@
       end
 
       it "returns follow suggestions that have profile images" do
-        user = create(:user)
-        tag = create(:tag)
-        user.follow(tag)
-
-        other_user = create(:user)
-        create(:article, user: other_user, tags: [tag.name])
-
         sign_in user
 
         get users_path(state: "follow_suggestions")
 
         response_user = response.parsed_body.first
         expect(response_user["profile_image_url"]).to eq(other_user.profile_image_url)
       end
+
+      it "returns the default suggested_users from SiteConfig if prefer_manual_suggested_users is true" do
+        allow(SiteConfig).to receive(:prefer_manual_suggested_users).and_return(true)
+
+        sign_in user
+
+        get users_path(state: "follow_suggestions")
+
+        expect(response).to have_http_status(:ok)
+        expect(response.parsed_body.first).to include(
+          "id" => suggested_user.id,
+          "name" => suggested_user.name,
+          "username" => suggested_user.username,
+          "summary" => suggested_user.summary,
+          "profile_image_url" => Images::Profile.call(suggested_user.profile_image_url, length: 90),
+          "following" => false,
+        )
+      end
```

Comment:
Thanks for adding these tests! 👏

Reply:
I second @vaidehijoshi -- thank you for adding these thorough tests! 🎉